core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3813,6 +3813,9 @@ components:
           description: Whether or not to delete intermediate output files when the
             workflow completes. See [Cromwell docs](https://cromwell.readthedocs.io/en/stable/wf_options/Google#google-pipelines-api-workflow-options)
             for more information.
+        executionPath:
+          type: string
+          description: The full GCS path indicating where execution took place.
```

Comment:
Just to confirm: this is only used in responses, not requests?

Reply:
Yes only in responses- it's left out of the `SubmissionRequest` case class